### ISSUE TITLE
when will zipkin-cpp support visual studio ?

### ISSUE THREAD

**Issue by MagicInTheWinds** [flier/zipkin-cpp#14]
It seems zipkin-cpp can not work on vc2015 yet :(

**Comment by flier** [comment]
I think it is not easy, I need to setup an environment to test with VS2015, I can only say it is theoretical feasibility :)

Maybe we could choose to use MingGW or other unix like system.

**Comment by codefromthecrypt** [comment]
Maybe someone can find a similar open source project that's able to build a
similar C project with windows via docker?

That or we could use appveyor with a similar recipe
https://github.com/openzipkin/zipkin-azure/pull/22

**Comment by flier** [comment]
Yes, appveyor has [VS2015 supports](https://www.appveyor.com/docs/build-environment/#build-worker-images), I will check it later
